ecl/eclplus/DumpHelper.cpp

```diff
@@ -17,6 +17,23 @@
 #include "jlib.hpp"
 #include "DumpHelper.ipp"
 
+static bool processExceptions(const IMultiException* excep)
+{
+    if (excep && excep->ordinality())
+    {
+        unsigned i = 0;
+        while (i < excep->ordinality())
+        {
+            StringBuffer msg;
+            excep->item(i).errorMessage(msg);
+            unsigned code = excep->item(i).errorCode();
+            printf("<Error><code>%d</code><message>%s</message></Error>\n", code, msg.str());
```

Comment:
That would be more sensible, but none of the existing code did (this was extracting existing code into a function)